editor/store/actions.js

```diff
@@ -220,8 +220,8 @@ export function replaceBlock( uid, block ) {
 function createOnMove( type ) {
 	return ( uids, rootUID ) => {
 		return {
+			uids: castArray( uids ),
```

Comment:
Same situation as https://github.com/WordPress/gutenberg/pull/6310/files#r183138903.

Reply:
This can't be updated since it's an inline function (no JS Docs).